rest/rest-cli/src/main/java/org/ow2/proactive_grid_cloud_portal/cli/cmd/sched/JobKeyValueTransformer.java

```diff
@@ -42,7 +42,7 @@
 
     public static Map<String, String> transformJsonStringToMap(String jsonString) {
         Map<String, String> jsonMap = Maps.newHashMap();
-        if (jsonString != null) {
+        if (jsonString != null && !jsonString.equals("undefined")) {
```

Comment:
To me, this piece of code could leverage nullPointerException. Please validate ``jsonString`` in an other conditional statement than ``!jsonString.equals("undefined")`` to prevent any unexpected behavior.